spec/factories/users.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :user do
+    name 'felix'
+    user_email 'felix@telia.com'
+    user_password 'fisk1234'
```

Comment:
Now that you have removed `user_email` and `user_password` you should get rid of them from the factory.